<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SinkComponent</class>
 <widget class="QWidget" name="SinkComponent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>227</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="componentLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>100</horstretch>
       <verstretch>100</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
     border: 2px solid gray;
     border-radius: 5px;
     margin-top: 1ex; /* leave space at the top for the title */
 }

 QGroupBox::title {
     subcontrol-origin: margin;
     subcontrol-position: top left; /* position at the top center */
     padding: 0 3px;
	left: 1em;
 }</string>
     </property>
     <property name="title">
      <string>Spout</string>
     </property>
     <widget class="QDial" name="dialAmount">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>15</y>
        <width>91</width>
        <height>111</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QDial" name="dialSeconds">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>15</y>
        <width>91</width>
        <height>101</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
      <property name="notchesVisible">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="labelVerb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>125</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinAmount">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>120</y>
        <width>59</width>
        <height>31</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelEvery">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>125</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>messages every</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinSeconds">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>120</y>
        <width>59</width>
        <height>31</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::PlusMinus</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>120</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="buttonStart">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>50</y>
        <width>97</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QStatusBar" name="statusBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>411</width>
        <height>26</height>
       </rect>
      </property>
      <property name="sizeGripEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
